package.json

```
@@ -8,6 +8,7 @@
     "build": "ncc build -o dist/setup src/setup-java.ts && ncc build -o dist/cleanup src/cleanup-java.ts",
     "format": "prettier --write \"{,!(node_modules)/**/}*.ts\"",
     "format-check": "prettier --check \"{,!(node_modules)/**/}*.ts\"",
+    "lint": "",
```

I think we should clarify why we need this
```suggestion
    "lint": "echo \"Fake command that does nothing. It is used in reusable workflows\"",
```